tests/dummy/app/pods/components/frost-create-form-with-scroll/template.hbs

```diff
@@ -1,18 +1,22 @@
-{{#frost-modal-input title='Test title' subtitle='Subtitle'}}
-{{!#frost-modal-input titleComponent='myTitle'}}
-
-  {{#perfect-scroll
+{{#frost-modal-input
+  title='Test title'
+  subtitle='Subtitle'
+  openButton='Open Modal with Perfect Scroll'
+  openButtonClasses='medium secondary'
+  modalName='my-ps-modal'}}
+  {{#block-slot slot 'form'}}
+    {{#perfect-scroll
```

Comment:
Do we need to make it optional?  I'd prefer that the scroll is always present, but only used when we exceed the max height (I think we had that working at some point?)